packages/api/src/cli/codegen/languages/typescript.ts

```diff
@@ -673,12 +673,17 @@ sdk.server('https://eu.api.example.com/v14');`)
       Object.entries(ops).forEach(([method, operation]: [HttpMethods, Operation]) => {
         methods.add(method);
 
-        const operationId = operation.getOperationId();
+        const operationId = operation.getOperationId({
+          // This `camelCase` option will clean up any weird characters that might be present in
+          // the `operationId` so as we don't break TS compilation with an invalid method accessor.
+          camelCase: true,
```

Comment:
I love how you already wrote support for this and you just forgot to enable it lmao

Reply:
lol me